validation-test/Evolution/test_class_add_convenience_init.swift

```diff
@@ -0,0 +1,20 @@
+// RUN: %target-resilience-test
+// REQUIRES: executable_test
+
+import StdlibUnittest
+import class_add_convenience_init
+
+var AddConvenienceInitTest = TestSuite("AddConvenienceInit")
+
+class Hobbit : AddConvenienceInit {
+  public let name: String = "Bilbo"
+}
+
+AddConvenienceInitTest.test("AddConvenienceInit") {
+  let t = Hobbit(z: 4)
```

Comment:
Can you make sure it's callable too, in the AFTER mode?

Reply:
The test file itself is linked with both the before and after versions so it cannot call a symbol that's only available in the after library (unless we use `@_weakLinked`, I guess). The new convenience init is called by other init though. Did you have something else in mind?

Reply:
I mean, we certainly don't want to break existing code, but we also want to make sure that the thing that was added is usable and backwards-deployable.